openshift-templates/rolebinding-group/template.yaml

```diff
@@ -0,0 +1,23 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: jenkins-pipeline-no-ocp-triggers
```

Comment:
Since this is a "generic" template, probably should use `${ROLE}` here as well?

Reply:
oh right, copy / paste error